pkg/kn/commands/service_create.go

```diff
@@ -69,11 +81,17 @@ func NewServiceCreateCommand(p *KnParams) *cobra.Command {
 			if err != nil {
 				return err
 			}
+			// check if --force flag is given
+			if force, err := cmd.Flags().GetBool("force"); err != nil {
+				return err
+			} else if force {
+				client.Services(namespace).Delete(args[0], &v1.DeleteOptions{})
+			}
 			_, err = client.Services(namespace).Create(&service)
 			if err != nil {
 				return err
 			}
-
+			fmt.Fprintf(cmd.OutOrStdout(), "Service '%s' is successfully created in namespace '%s'.\n", args[0], namespace)
```

Comment:
`Service '%s' successfully created in namespace '%s'` (i.e. without "is")

Reply:
If we replaced it, say so in the string too